apps/docs/guide/getting-started.md

```diff
@@ -29,131 +30,112 @@ $ pnpm install @rtco/client
 ```sh [bun]
 $ bun add @rtco/client
 ```
+
 :::
 
+## Setup
 
-## Call Example
+### Basic Setup
 
-The following example shows how to connect two peers and share audio/video/data between them:
+```js
+import { Artico } from "@rtco/client";
+const rtco = new Artico();
+```
 
-#### Peer 1
+### Setup with Custom ID
 
-```ts
-import { Artico } from "@rtco/client";
+You can request a specific peer ID. If you do, Artico will attempt to register such ID with the signaling server. If not available, Artico will emit an Error with the message "invalid-id".
 
-const rtco = new Artico();
+:::Warning: It is recommended that you let Artico assign you a random ID, since IDs should be universally unique within Artico's signaling network.:::
```

Comment:
Replace this line with:

```
::: tip
It is recommended that you let Artico assign you a random ID, since IDs should be universally unique within Artico's signaling network.
:::
```

Reply:
Did the server update just now? I am getting a "Error connecting to signalling server". Now..

Reply:
I've updated those in the latest commit